packages/suite/src/components/suite/OverflowItems.tsx

```diff
@@ -0,0 +1,134 @@
+import { ElementType, ReactNode, useCallback, useEffect, useRef, useState } from 'react';
+import { usePrevious, useShallowCompareEffect, useUpdateEffect } from 'react-use';
+import { useSelector } from 'src/hooks/suite';
+
+type Item = {
+    removeOrder?: number;
+    [key: string]: any;
+};
+
+interface OverflowItemsProps<T> {
+    wrapper?: ElementType;
+    items: T[];
+    visibleItemRenderer: (item: T, index: number) => ReactNode;
```

Comment:
I believe it would be better to avoid this renderer pattern since I believe this OverflowItems will always have an array of buttons which are folded into a dropdown. So you can just define [those](https://github.com/trezor/trezor-suite/blob/2726578bb5e95c8b96c3cef603deb511d1f15e6b/packages/suite/src/views/wallet/transactions/TradeBox/TradeBoxMenu.tsx#L93) right here, in `OverflowItems` and add an `onClick` and `data-test` fields to the `Item`

Reply:
I wanted to make a more generic component to accept any elements, not just buttons

Reply:
I don't think we will have any use cases for other than buttons anytime soon though 🤔

Reply:
Isn't that a bit of shame to not have a generic one and to make additional dependency which can't be changed from outside? seems like an antipattern to me when the job is already done.

Reply:
True, engineering-wise it's not a bad solution at all! All I am saying is that I don't think we are going to have any other cases other than those buttons in the `TradeBox` and `AppNavigation` + Suite's design is about to change. Sometimes it's better to have a less generic, simpler solution to avoid having unnecessary complications later down the line. Since it's already done – leave it there :)